src/ape_test/runners.py

```diff
@@ -30,6 +34,61 @@ def _network_choice(self) -> str:
         # The option the user providers via --network (or the default).
         return self.config.getoption("network")
 
+    # @ape_cli_context()
+    def pytest_exception_interact(self, report, call):
+        """
+        A `-I` option triggers when an exception is raised which can be interactively handeled.
+        Outputs the full repr of the failed test and opens an interactive shell using `ape console`.
+        """
+
+        if self.config.getoption("interactive") and report.failed:
+
+            capman = self.config.pluginmanager.get_plugin("capturemanager")
+            if capman:
+                capman.suspend_global_capture(in_=True)
+
+            # find the last traceback frame within the active project
+            traceback = call.excinfo.traceback[-1]
+            for tb_frame in call.excinfo.traceback[::-1]:
+                try:
+                    Path(tb_frame.path).relative_to(self.project.path)
+                    traceback = tb_frame
+                    click.echo()
```

Comment:
You _could_ replace this `click.echo()` with a `\n` at the start of the next one.

Now that we are not using the logger here, it should work out better.